### PR TITLE
Add github action

### PR DESCRIPTION
**Pull request by amai2012**

### CODE REVIEW

File: .github/workflows/nodejs.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+        node-version: ${{ matrix.node-version }}
+    
+    - name: Install npm
+      run: npm install
+    
+    - name: Build
+      run: node make
+    
+    - name: Run lint
+      run: npm run lint
```
Comment (danmar) [inline .github/workflows/nodejs.yml:35]: curious; what exactly does this "lint" step do?

**Review by danmar — COMMENTED** [review]
I am not sure what this node stuff is. Is it something that is used by github actions?

**Comment by amai2012** [comment]
> I am not sure what this node stuff is. Is it something that is used by github actions?

I don't know either :blush:
I was just following the instructions from https://github.com/danmar/cppcheck-htdocs/blob/master/README.md :smirk: So at least instructions and actions are in sync...

**Comment by danmar** [comment]
aha.. ok good.

**Pull request merged**